resolwe_bio/processes/alignment/hisat2.yml

```diff
@@ -8,7 +8,7 @@
   requirements:
     expression-engine: jinja
   data_name: "Alignment ({{ reads|sample_name|default('?') }})"
-  version: 1.0.7
+  version: 1.0.8
```

Comment:
We should discuss which number in version to increase for which changes.

This change introduces new features, but is backward compatible. So I think middle number should be increased.

cc @tjanez, @mstajdohar, @jkokosar